docs/UserGuide.md

```diff
@@ -58,121 +33,47 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
 
 </div>
 
-### Viewing help : `help`
-
-Shows a message explaning how to access the help page.
-
-![help message](images/helpMessage.png)
-
-Format: `help`
+### Adding a client : `add`
 
+Adds a person to the list with a note beside their name. 
 
-### Adding a person: `add`
-
-Adds a person to the address book.
-
-Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​`
-
-<div markdown="span" class="alert alert-primary">:bulb: **Tip:**
-A person can have any number of tags (including 0)
-</div>
+Format: `add n/NAME o/NOTE`
```

Comment:
Perhaps this could be Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [o/NOTE] [t/TAG]…​`

Reply:
I think we should change this along the way for v1.2. Since we are not sure as of yet if we want to remove the fields or add some fields

Reply:
For now, for v1.2, I think we would try to minimise the amount of fields to delete, so we would add in NOTE and keep the rest. However, this may change in v1.3/1.4, so we can update the UG then.

Reply:
We can add back the "Tip" for tag if you are okay with the above change.